### PR TITLE
feat(an): New review flow

### PR DESCRIPTION
**Pull request by sigruntg**
# New review flow

https://app.clickup.com/t/1f36kmz
https://app.clickup.com/t/1f36ktk
https://app.clickup.com/t/1f36m4r

## What

Updateing review to one page that switches between review, add documents and overview.

## Why

Specify why you need to achieve this

## Screenshots / Gifs

image
image
image

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/accident-notification/src/lib/messages/thirdPartyComments.ts [hunk 1]
```diff
@@ -2,10 +2,16 @@ import { defineMessages } from 'react-intl'
...
-      description: 'Accident description from applicant',
+    title: {
+      id: 'an.application:thirdPartyComment.general.title',
+      defaultMessage: 'Bæta við athugasemd',
+      description: 'Third party comment title',
+    },
+    description: {
+      id: 'an.application:thirdPartyComment.general.description',
+      defaultMessage:
+        'Ef það eru einhverjar upplýsingar varðandi tilkynninguna sem þú vilt setja útá eða auka upplýsingar varðandi slysið sem þú vilt koma á framfæri þá er hægt að gera það hér að neðan.',
```
Comment (Thorrik58) [inline libs/application/templates/accident-notification/src/lib/messages/thirdPartyComments.ts:13]:
```suggestion
        'Ef það eru einhverjar upplýsingar varðandi tilkynninguna sem þú vilt setja útá eða auka upplýsingar varðandi slysið sem þú vilt koma á framfæri þá er hægt að gera það hér að neðan',
```

**Review by Thorrik58 — CHANGES_REQUESTED** [review]
Just some minor comments

File: libs/application/templates/accident-notification/src/utils/getInjuredPersonInformation.spec.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { FormValue } from '@island.is/application/core'
+import { getInjuredPersonInformation } from './getInjuredPersonInformation'
+
+describe('getInjuredPersonInformation', () => {
+  const injuredPersonInformation: FormValue = {
+    injuredPersonInformation: {
+      email: 'sigrun@sendiradid.is',
```
[the remark this answers is not in the document]
Comment (sigruntg) [reply libs/application/templates/accident-notification/src/utils/getInjuredPersonInformation.spec.ts:7]: I'm always looking for new friends 🤓

File: libs/application/templates/accident-notification/src/fields/InReviewSteps/index.tsx [hunk 3]
```diff
@@ -1,74 +1,86 @@
...
+}) => {
   const { formatMessage } = useLocale()
 
-  const [submitApplication, { loading: loadingSubmit }] = useMutation(
-    SUBMIT_APPLICATION,
-    {
-      onError: (e) => console.error(e.message),
-    },
-  )
+  const showMissingDocumentsMessage = (answers: FormValue) => {
```
[the remark this answers is not in the document]
Comment (sigruntg) [reply libs/application/templates/accident-notification/src/fields/InReviewSteps/index.tsx:49]: I don't think we need this anymore, I will only check if it has missing documents.

File: libs/application/templates/accident-notification/src/fields/InReviewSteps/index.tsx [hunk 4]
```diff
@@ -80,20 +92,22 @@ export const InReviewSteps: FC<FieldBaseProps> = ({ application, refetch }) => {
...
       action: {
         title: formatMessage(inReview.action.representative.title),
         description: formatMessage(inReview.action.representative.description),
         actionButtonTitle: formatMessage(
           inReview.action.representative.actionButtonTitle,
         ),
       },
+      visible:
+        !isHomeActivitiesAccident(application.answers) ||
+        !isRepresentativeOfCompanyOrInstitute(application.answers),
```
[the remark this answers is not in the document]
Comment (sigruntg) [reply libs/application/templates/accident-notification/src/fields/InReviewSteps/index.tsx:110]: No, if you are not a representitive of company and the injured and if it was not home activity accident it should return false and you will not see this.

**Pull request merged**